pkg/operator/configobservation/network/observe_network.go

```diff
@@ -14,39 +14,32 @@ import (
 
 // ObserveRestrictedCIDRs watches the network configuration and updates the
 // RestrictedEndpointsAdmission controller config.
-func ObserveRestrictedCIDRs(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
-	listers := genericListers.(configobservation.Listers)
+func ObserveRestrictedCIDRs(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (ret map[string]interface{}, _ []error) {
+	configPath := []string{"admission", "pluginConfig", "network.openshift.io/RestrictedEndpointsAdmission", "configuration"}
+	defer func() {
+		ret = configobserver.Pruned(ret, configPath)
+	}()
 
+	listers := genericListers.(configobservation.Listers)
 	var errs []error
-	configPath := []string{"admission", "pluginConfig", "network.openshift.io/RestrictedEndpointsAdmission", "configuration"}
 
 	admissionControllerConfig := unstructured.Unstructured{}
-	prev, ok, err := unstructured.NestedMap(existingConfig, configPath...)
-	if err != nil {
-		errs = append(errs, err)
-	}
-	if ok && err == nil {
-		// If we ever bump the API version, we'll have to do conversion here.
-		admissionControllerConfig.SetUnstructuredContent(prev)
-	}
 	admissionControllerConfig.SetAPIVersion("network.openshift.io/v1")
 	admissionControllerConfig.SetKind("RestrictedEndpointsAdmissionConfig")
 
 	clusterCIDRs, err := network.GetClusterCIDRs(listers.NetworkLister, recorder)
 	if err != nil {
-		errs = append(errs, err)
```

Comment:
While I do prefer to sum errors, the reality is that a cluster with a missing ServiceCIDR and ClusterCIDR is unlikely, and not a user error, so I'll let this one go :-)